content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -19,6 +19,12 @@ accepting traffic. A Pod is considered ready when all of its Containers are read
 One use of this signal is to control which Pods are used as backends for Services.
 When a Pod is not ready, it is removed from Service load balancers.
 
+The kubelet uses startup probes to know when a Container application has started.
+If such a probe is configured, it disables liveness and readiness checks until
+it succeeds, making sure those probes don't interfere with the application startup.
```

Comment:
nit: If a startup probe is configured, the kubelet(?) disables liveness and readiness checks until the container application successfully starts?
nit: line 25, What is `It`?

Reply:
Alright, there is a lot of IT... could you help me rephrase in proper English (I'm French...)?

The kubelet uses startup probes to know when a Container application has started.
If such a probe is configured, it (kubelet) disables liveness and readiness checks until it (startupProbe) succeeds, making sure those probes don't interfere with the application startup.
It (this technique) can be used to adopt liveness checks on slow starting containers, avoiding them getting killed by the kubelet before they are up and running.

Reply:
@matthyx  Update ```It can be used to adopt liveness checks ``` to ```This can be used to adopt liveness checks ```  on Line 25.